vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/factory_object_mapping.go

```diff
@@ -78,7 +78,11 @@ func NewObjectMappingFactory(clientAccessFactory ClientAccessFactory) ObjectMapp
 // the underlying Scheme supports Unstructured. The mapper will return converters that can
 // convert versioned types to unstructured and back.
 // It can return an error and the best effort unstructured mapper and typer.
-func (f *ring1Factory) objectLoader() (meta.RESTMapper, runtime.ObjectTyper, error) {
+func (f *ring1Factory) objectLoader(local bool) (meta.RESTMapper, runtime.ObjectTyper, error) {
+	if local {
+		return legacyscheme.Registry.RESTMapper(), legacyscheme.Scheme, nil
```

Comment:
We need to remove all trace of legacyscheme and other hardcoded schemes from our code.  This isn't an acceptable solution.